UnitsNet.Tests/CustomCode/VolumeFlowTests.cs

```diff
@@ -131,6 +131,8 @@ public class VolumeFlowTests : VolumeFlowTestsBase
 
         protected override double CubicMillimetersPerSecondInOneCubicMeterPerSecond => 1e9;
 
+        protected override double CubicCentimetersPerMinuteInOneCubicMeterPerSecond => 59988002.39;
```

Comment:
I think it would be more appropriate to use the actual value here which according to [this](https://www.convertunits.com/from/cubic+centimeters+per+minute/to/cubic+metre/second) is exactly 60000000.
I think you could add more 6's to coefficient or lower the corresponding tolerance value if the test is failing.